src/pages/patientView/clinicalInformation/PatientViewPageStore.ts

```diff
@@ -1696,6 +1788,173 @@ export class PatientViewPageStore {
         []
     );
 
+    readonly oncoKbDataForOncoprint = remoteData<IOncoKbData | Error>(
+        {
+            await: () => [this.mutationData, this.oncoKbAnnotatedGenes],
+            invoke: async () =>
+                fetchOncoKbDataForOncoprint(
+                    this.oncoKbAnnotatedGenes,
+                    this.mutationData
+                ),
+            onError: (err: Error) => {
+                // fail silently, leave the error handling responsibility to the data consumer
+            },
+        },
+        ONCOKB_DEFAULT
+    );
+
+    readonly cnaOncoKbDataForOncoprint = remoteData<IOncoKbData | Error>(
+        {
+            await: () => [
+                this.clinicalDataForSamples,
+                this.studiesForSamplesWithoutCancerTypeClinicalData,
+                this.samplesWithoutCancerTypeClinicalData,
+                this.oncoKbAnnotatedGenes,
+                this.molecularData,
+                this.molecularProfileIdToMolecularProfile,
+            ],
+            invoke: async () =>
+                fetchCnaOncoKbDataForOncoprint(
+                    this.oncoKbAnnotatedGenes,
+                    this.molecularData,
+                    this.molecularProfileIdToMolecularProfile
+                ),
+        },
+        ONCOKB_DEFAULT
+    );
+
+    readonly getPutativeDriverInfo = remoteData({
+        await: () => [
+            this.getOncoKbMutationAnnotationForOncoprint,
+            this.isHotspotForOncoprint,
+        ],
+        invoke: () => {
+            return Promise.resolve((mutation: Mutation): {
+                oncoKb: string;
+                hotspots: boolean;
+                cbioportalCount: boolean;
+                cosmicCount: boolean;
+                customDriverBinary: boolean;
+                customDriverTier?: string;
+            } => {
+                const getOncoKbMutationAnnotationForOncoprint = this
+                    .getOncoKbMutationAnnotationForOncoprint.result!;
+                const oncoKbDatum:
+                    | IndicatorQueryResp
```

Comment:
wow.  why is this type so wide open?

Reply:
Cannot remember why, this is partially copied from https://github.com/cBioPortal/cbioportal-frontend/blob/master/src/pages/resultsView/ResultsViewPageStore.ts#L4176